src/main/kotlin/uoslife/servermeeting/user/entity/User.kt

```diff
@@ -20,7 +20,7 @@ class User(
     var kakaoTalkId: String = "",
     @OneToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "tos_id", foreignKey = ForeignKey(ConstraintMode.CONSTRAINT))
-    var tos: Tos = Tos(),
+    var tos: Tos? = null,
```

Comment:
저번 회의에서 얘기했듯 굳이 가지고 있을 필요 없는 컬럼이라 삭제해도 될 것 같습니다.
제 PR에서는 다 삭제했습니다.
```suggestion
    var tos: Tos? = null,
```

Reply:
머지할 때 충돌나면 삭제하시면 됩니다.